frontend/src/Components/EventsCard.jsx

```diff
@@ -4,18 +4,26 @@ import React from 'react';
 const EventsSearchCard = (props) => {
```

Comment:
could you please rename this file to match the component name. not a breaker but good for consistency